app/src/main/java/com/github/pockethub/android/ui/repo/RepositoryContributorsFragment.java

```diff
@@ -106,8 +110,9 @@ public void onListItemClick(ListView l, View v, int position, long id) {
                 .getUser(contributor.login())
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .compose(this.<User>bindToLifecycle())
-                .subscribe(user -> startActivity(UserViewActivity.createIntent(user)));
+                .compose(this.bindToLifecycle())
+                .subscribe(
+                        response -> startActivity(UserViewActivity.createIntent(response.body())));
```

Comment:
Shouldn't response -> be on the same line as subscribe to keep consistent with coding style e.g. here?

I.e.

.subscribe(response ->
            `startActivity(UserViewActivity.createIntent(response.body())));`